### ISSUE TITLE
How do check if a counter is different than X?

### ISSUE THREAD

**Issue by riahc3** [NagiosEnterprises/ncpa#855]
Hello I want to check a performance counter

./check_ncpa.py -H localhost -t 'fkoldjghlkdfjhgkdf' -M 'windowscounters/acounter'

But I want to throw a critical if it is different than the number X

How can I do this?

Thank you

**Comment by riahc3** [comment]
So

./check_ncpa.py -H localhost -t 'fkoldjghlkdfjhgkdf' -M 'windowscounters/acounter' -c 10:10

Would mean anything above or below 10, is critical.

While on the subject:

./check_ncpa.py -H localhost -t 'fkoldjghlkdfjhgkdf' -M 'windowscounters/acounter' -w :10 -c 5:

This means if its less than 10 (but more than 5) it is a warning. If it is less than 5, it is a critical. Correct?

**Comment by ericloyd** [comment]
Yes, but it may depend on whether the code interprets the warning threshold before or after the critical threshold and whether or not it checks the other once it's checked the first.  You're entering a somewhat undefined territory as far as behavior versus specification here.

**Comment by riahc3** [comment]
Excuse me. I made a VERY IMPORTANT TYPO.

Edited.
